### PR TITLE
Cql4bonnie 530

### PR DESCRIPTION
**Pull request by edeyoung**
@holmesie @mulcahyk 
why are these changes necessary? they don't seem to break anything, but i was able to use the HDS changes without these.

If necessary, I give my approval for this PR (I'm posting on Kristian's behalf).

Should we wait until the front-end code perhaps needs this to make any changes here?

### CODE REVIEW

**Comment by jbradl11** [comment]
@edeyoung Without these changes there is no way to grab the observation "population" in Bonnie. HDS will work and the measure will load into bonnie with no problems without these changes, however if we want to display or use observations in bonnie we need these changes. 

This is the code that grabs the parsed observation from HQMF and saves it to the CQLMeasure Model in the MongoDB. This pull request looks good to me, and I believe this is the first step for us to get Observations working on the front-end.

**Review by jbradl11 — APPROVED** [review]

**Comment by edeyoung** [comment]
test failure is the expected test failure that we have been seeing in bonnie bundler and have it on our list to explore.

File: lib/measures/loading/loader.rb [hunk 1]
```diff
@@ -88,6 +88,7 @@ def self.load_hqmf_cql_model_json(json, user, measure_oids, elm, cql)
       measure.measure_period = json["measure_period"]
       measure.population_criteria = json["population_criteria"]
       measure.populations_cql_map = json["populations_cql_map"]
+      measure.observations = json["observations"]
```
Comment (jbradl11) [inline lib/measures/loading/loader.rb:123]: After stepping into the code, this looks like its not returning the correct values. It looks like you want to use 
json[:observations]

json[:observations]  returns ["Decision To Admit"]
json['observations'] returns nil

Comment (holmesie) [reply lib/measures/loading/loader.rb:123]: What version of Ruby are you running? If json['observations'] (or more accurately json["observations"]) doesn't work, does json["populations_cql_map"] even work?

Comment (jbradl11) [reply lib/measures/loading/loader.rb:123]: I am running 2.2.2. And no it looks like json["populations_cql_map"] returns nil as well. However json[:populations_cql_map] returns the expected values.

Comment (jbradl11) [reply lib/measures/loading/loader.rb:123]: Never mind there is a line of code that changes all keys to strings after where I put the break point.

**Review by holmesie — APPROVED** [review]

**Review by c-monkey — APPROVED** [review]
how did we forget this? GAH!

**Pull request merged**